Examples/StereoKitTest/Tests/TestNodes.cs

```diff
@@ -187,7 +197,12 @@ static void RecursiveTraversal(ModelNode node, int depth = 0)
 	}
 	/// :End:
 
-	public void Shutdown() { }
+	public void Shutdown() 
+	{
+		// Verify ModelNode and Ray, Ray and ModelNode intersections.
+		Tests.Test(TestModelNodeRayIntersections);
```

Comment:
Is there a reason this is being done in `Shutdown` instead of at the end of `Initialize` with the rest of the `Test` calls? If not, it would be good to group those together :)